src/providers/gitlab.js

```diff
@@ -0,0 +1,28 @@
+const gitlabConfig = {
+  name: 'gitlab',
+  selectors: {
+    // Row in file list, file view header
+    row: 'table[data-qa-selector="file_tree_table"] tr, .file-header-content',
+    // Cell in file list, file view header, readme header
+    filename: 'td.tree-item-file-name, .file-title-name, .gl-link',
+    // Any icon not contained in a button
+    icon: 'svg:not(.gl-button-icon)',
```

Comment:
I'd rather keep the selectors contained to only match within what we identified as a row. This way if gitlab decides to add svgs elsewhere in the page we are less likely to break.

Maybe a simple[ descendant selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Descendant_combinator) might do the trick here?
```suggestion
    icon: '.tree-item svg',
```

Reply:
For this one, I could not find a more specific selector that would be able to target the icon in the file headers (both viewing a file, and the readme on the repo's homepage).  The only way I can think to specify it more would be to remove the icons from those other views.

Reply:
To specify, the reason for doing `:not(.gl-button-icon)` is to prevent matching the copy button icon in the file headers btw.